R/common-null-pointer.R

```diff
@@ -0,0 +1,40 @@
+#' @title 
```

Comment:
Where does the name come from? something like `countMissingData` makes more sense to me.